packages/rx_bloc_cli/mason_templates/bricks/lib_dev_menu/__brick__/bin/remove_alice.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+
+# NOTE: this script should be executed in fastlane only
+
+if flutter pub deps | grep -q alice ; then flutter pub remove alice; fi
+
+if [ -e lib/lib_dev_menu ]; then rm -r lib/lib_dev_menu; fi
+if [ -e lib/main_dev.dart ]; then rm lib/main_dev.dart; fi
+if [ -e lib/main_staging.dart ]; then rm lib/main_staging.dart; fi
+if [ -e lib/alice_instance.dart ]; then rm lib/alice_instance.dart; fi
```

Comment:
See the comment for the recommended location for that file (lib/alice_instance.dart)